src/suite-design-guide/general-principles.rst

```diff
@@ -332,15 +332,15 @@ queues*, respectively.
 Runahead Limiting
 ^^^^^^^^^^^^^^^^^
 
-By default Cylc allows a maximum of three cycle points to be active at the same
+By default Cylc allows a maximum of five cycle points to be active at the same
 time, but this value is configurable:
 
 .. code-block:: cylc
 
    [scheduling]
        initial cycle point = 2020-01-01T00
        # Don't allow any cycle interleaving:
-       max active cycle points = 1
+       runahead limit = 1
```

Comment:
That should be `runahead limit = P1`. I already have a PR up for that #161, so maybe you can merge that into this one

Reply:
Derp, sorry should have spotted that one.

Reply:
```suggestion
       runahead limit = P1
```